default/python/AI/AIDependencies.py

```diff
@@ -310,8 +310,7 @@ class Tags:
 # <editor-fold desc="Supply techs">
 supply_range_techs = {
     "CON_ORBITAL_CON": 1,
-    "CON_CONTGRAV_ARCH": 1,
-    "CON_GAL_INFRA": 1,
```

Comment:
@Cjkjvfnby any concerns abou these Python AI changes?

Reply:
I'm going to merge this now as I want it in the next test build, so if there are issues with the AI, let me know or fix them later...